hydra/test_utils/test_utils.py

```diff
@@ -242,17 +247,18 @@ def verify_dir_outputs(
     assert job_return.task_name is not None
     assert job_return.hydra_cfg is not None
 
-    assert os.path.exists(
-        os.path.join(job_return.working_dir, job_return.task_name + ".log")
-    )
-    hydra_dir = os.path.join(
-        job_return.working_dir, job_return.hydra_cfg.hydra.output_subdir
-    )
-    assert os.path.exists(os.path.join(hydra_dir, "config.yaml"))
-    assert os.path.exists(os.path.join(hydra_dir, "overrides.yaml"))
-    assert OmegaConf.load(
-        os.path.join(hydra_dir, "overrides.yaml")
-    ) == OmegaConf.create(overrides or [])
+    if not remote:
```

Comment:
this entire things feels like a hack to me.
the problem is that you do not have the output directory. your workaround is to not do anything with it.